client/src/routes/automation/esidomGenerator.ts

```diff
@@ -58,12 +63,21 @@ interface BlocksGenerator {
     action: (blk: Block) => void;
     color_picker: (blk: Block) => void;
     color_rgb: (blk: Block) => void;
+    binary_sensor: (blk: Block) => void;
+    person: (blk: Block) => void;
+    weather: (blk: Block) => void;
+    zwave: (blk: Block) => void;
+    sensor: (blk: Block) => void;
+    light: (blk: Block) => void;
+    automation: (blk: Block) => void;
+    switch: (blk: Block) => void;
+    media_player: (blk: Block) => void;
     scrub: (blk: Block, code: string, opt_thisOnly: string) => void
     jsonInit:(a: Block)=> void;
 }
 
```

Comment:
Is there a way to loop affectation ?

Reply:
Those are not affectations, they are the type declarations of each members of the interface.

Reply:
Can't loop then ? Always the same type declaration

Reply:
No, I don't think we can loop here.